CoreLayersView/src/au/gov/asd/tac/constellation/views/layers/state/LayersViewConcept.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2010-2020 Australian Signals Directorate
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.views.layers.state;
+
+import au.gov.asd.tac.constellation.graph.GraphElementType;
+import au.gov.asd.tac.constellation.graph.schema.attribute.SchemaAttribute;
+import au.gov.asd.tac.constellation.graph.schema.concept.SchemaConcept;
+import au.gov.asd.tac.constellation.graph.schema.concept.SchemaConcept.ConstellationViewsConcept;
+import au.gov.asd.tac.constellation.views.layers.layer.LayerDescription;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import org.openide.util.lookup.ServiceProvider;
+
+/**
+ * Attribute provider for attributes specific to the Layers View.
+ *
+ * @author aldebaran30701
+ */
+@ServiceProvider(service = SchemaConcept.class)
+public class LayersViewConcept extends SchemaConcept {
+
+    private static final LayersViewState DEFAULT_STATE = new LayersViewState(
```

Comment:
It might make more sense to move this to the `LayerViewState` class.

Reply:
I can see how that could be a beneficial change, and I was thinking the same thing myself. With the current implementation I've just copied the same structure as AnalyticViewState and -Concept. I may be wrong but a change could make it inconsistent with the rest of the state implementations.

Reply:
Haha i'm pretty sure there's almost no consistency to the state implementations right now - i'll let you decide what's better and we'll go with that for now.

Reply:
I've implemented this in the current commit.